src/main/java/org/bitsofinfo/hazelcast/discovery/docker/swarm/SwarmDiscoveryUtil.java

```diff
@@ -48,56 +48,34 @@
     private static final int SOCKET_TIMEOUT_MILLIS = (int) TimeUnit.SECONDS.toMillis(1);
     private static final int SOCKET_BACKLOG_LENGTH = 100;
 
-    private Set<String> dockerNetworkNames = new TreeSet<>(String.CASE_INSENSITIVE_ORDER);
-    private Map<String, String> dockerServiceLabels = new TreeMap<>(String.CASE_INSENSITIVE_ORDER);
-    private Set<String> dockerServiceNames = new TreeSet<>(String.CASE_INSENSITIVE_ORDER);
+    private final Set<String> dockerNetworkNames = new TreeSet<>(String.CASE_INSENSITIVE_ORDER);
+    private final Map<String, String> dockerServiceLabels = new TreeMap<>(String.CASE_INSENSITIVE_ORDER);
+    private final Set<String> dockerServiceNames = new TreeSet<>(String.CASE_INSENSITIVE_ORDER);
 
-    private String rawDockerNetworkNames = null;
-    private String rawDockerServiceLabels = null;
-    private String rawDockerServiceNames = null;
+    private final String rawDockerNetworkNames;
+    private final String rawDockerServiceLabels;
+    private final String rawDockerServiceNames;
 
-    private Integer hazelcastPeerPort = 5701;
+    private final Integer hazelcastPeerPort;
 
     private DiscoveredContainer myContainer = null;
     private Address myAddress = null;
 
-    private boolean bindSocketChannel = true;
+    private final boolean bindSocketChannel;
     private ServerSocketChannel serverSocketChannel = null;
 
-    private URI swarmMgrUri = null;
-    private boolean skipVerifySsl = false;
+    private final URI swarmMgrUri;
+    private final boolean skipVerifySsl;
 
-    private boolean logAllServiceNamesOnFailedDiscovery = false;
-    private boolean strictDockerServiceNameComparison = false;
+    private final boolean logAllServiceNamesOnFailedDiscovery;
+    private final boolean strictDockerServiceNameComparison;
 
     // Since SwarmDiscoveryUtil is used by several components
     // the context lets us distinguish instances in logs
     private String context = null;
 
     private ILogger logger = Logger.getLogger(SwarmDiscoveryUtil.class);
 
-    public SwarmDiscoveryUtil(String context,
```

Comment:
Do not alter/remove any public signatures/constructors. This could break others using this library in use-cases specific to them such as programmatically configuring the stack.

Reply:
I don't believe that constructor has actually changed recently.  I would, however, love to get rid of "context" as it makes log messages longer than would otherwise be the case.  Of course it might be that from a support perspective having SwarmDiscoveryUtil[CallingClass] in the log message has proven helpful.

Reply:
Leave the context.

Don't have time at the moment to go through each these diffs in detail but perhaps they are presented in confusing manner in this UI. I just see red removal here of an entire public constructor. 

In any case, I don't mind formatting changes but do not alter or remove any pre-existing public method signatures or public constructors, regardless of if this library is internally calling it or not. Other users of this library could be calling these methods/constructors for other uses and this will maintain backwards compatibility.